app/components/bookings/create/create-booking.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-header>
+  <ion-navbar>
+    <ion-title>{{ _workpackage.stringId }} - {{ _workpackage.name }}</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+    <button accent floating block full *ngIf="!_bookingSession" (click)="createBookingSession()" >
+      <ion-icon name="ios-stopwatch-outline"></ion-icon>Livebuchung starten
+    </button>
+    <button accent floating block full *ngIf="_bookingSession" (click)="checkoutBookingSession()" >
+      <ion-icon name="ios-checkmark"></ion-icon>Checkout Session
+    </button>
+  <form [ngFormModel]="authForm" (ngSubmit)="onSubmit(authForm)">
+    <ion-item>
+      <ion-label>Datum</ion-label>
+      <ion-datetime displayFormat="DD.MM.YYYY" pickerFormat="DD MM YYYY" max="{{myDate}}"
+                    [(ngModel)]="myDate" [ngFormControl]="date" required></ion-datetime>
+    </ion-item>
+    <ion-item>
+      <ion-label>Aufwand</ion-label>
+      <ion-datetime displayFormat="HH:mm" pickerFormat="HH mm"
+                    [(ngModel)]="myEffort" [ngFormControl]="effort" (ionChange)="refreshEtc()"  required></ion-datetime>
+    </ion-item>
+    <ion-item class="item">
+      <ion-label floating >Beschreibung</ion-label>
+      <ion-textarea type="text" [ngFormControl]="description" required></ion-textarea>
+    </ion-item>
+    <ion-item>
+      <ion-label>ETC-aktuell:</ion-label>
+      <ion-label>{{ _workpackage.etc}}</ion-label>
```

Comment:
Ionic has (as far as I know) no way to make the picker >24 hours. This has to be done in an workaround